src/pages/Checkout/CheckoutBetaPage.vue

```diff
@@ -82,4 +87,9 @@ export default {
 .page-content {
 	padding: 1.625rem 0;
 }
+
+// Hide Basket Bar
+.basket-bar {
+	display: none;
+}
```

Comment:
@BoulderBrains I think this is all it takes to cover FECK-138... will add notes there

Reply:
Nice